airbyte-integrations/connectors/source-stock-ticker-api/source_stock_ticker_api/source.py

```diff
@@ -0,0 +1,107 @@
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+
+import requests
+import logging
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+from airbyte_cdk.sources.streams.http.auth import NoAuth
+
+from datetime import datetime, timedelta
+from airbyte_cdk.sources.streams import IncrementalMixin
+
+logger = logging.getLogger("airbyte")
+
+
+class ExchangeRates(HttpStream, IncrementalMixin):
+    url_base = "http://api.exchangeratesapi.io/"
+    cursor_field = "date"
+    primary_key = "date"
```

Comment:
I bet this won't work. You can have multiple records with the same date, so it would not uniquely identify any of those records